syntaxes/racket.tmLanguage.json

```diff
@@ -166,7 +166,7 @@
read-cell\\? | thread-dead-evt | thread-dead\\? | thread-group\\? | thread-receive | thread-receive-evt | thread-resume | thread-resume-evt | thread-rewind-receive | thread-running\\? | thread-send | thread-suspend | thread-suspend-evt | thread-try-receive | thread-wait | thread\\/suspend-to-kill | thread\\? | time-apply | touch | true | truncate | udp-addresses | udp-bind! | udp-bound\\? | udp-close | udp-connect! | udp-connected\\? | udp-multicast-interface | udp-multicast-join-group! | udp-multicast-leave-group! | udp-multicast-loopback\\? | udp-multicast-set-interface! | udp-multicast-set-loopback! | udp-multicast-set-ttl! | udp-multicast-ttl | udp-open-socket | udp-receive! | udp-receive!\\* | udp-receive!-evt | udp-receive!\\/enable-break | udp-receive-ready-evt | udp-send | udp-send\\* | udp-send-evt | udp-send-ready-evt | udp-send-to | udp-send-to\\* | udp-send-to-evt | udp-send-to\\/enable-break | udp-send\\/enable-break | udp\\? | unbox | uncaught-exception-handler | unit\\? | unquoted-printing-string | unquoted-printing-string-value | unquoted-printing-string\\? | unspecified-dom | unsupplied-arg\\? | use-collection-link-paths | use-compiled-file-check | use-compiled-file-paths | use-user-specific-search-paths | user-execute-bit | user-read-bit | user-write-bit | value-blame | value-contract | values | variable-reference->empty-namespace | variable-reference->module-base-phase | variable-reference->module-declaration-inspector | variable-reference->module-path-index | variable-reference->module-source | variable-reference->namespace | variable-reference->phase | variable-reference->resolved-module-path | variable-reference-constant\\? | variable-reference\\? | vector | vector->immutable-vector | vector->list | vector->pseudo-random-generator | vector->pseudo-random-generator! | vector->values | vector-append | vector-argmax | vector-argmin | vector-cas! | vector-copy | vector-copy! | vector-count | vector-drop | vector-drop-right | vector-fill! | vector-filter | vector-filter-not | vector-immutable | vector-length | vector-map | vector-map! | vector-member | vector-memq | vector-memv | vector-ref | vector-set! | vector-set\\*! | vector-set-performance-stats! | vector-split-at | vector-split-at-right | vector-take | vector-take-right | vector\\? | version | void | void\\? | weak-box-value | weak-box\\? | weak-set | weak-seteq | weak-seteqv | will-execute | will-executor\\? | will-register | will-try-execute | with-input-from-bytes | with-input-from-string | with-output-to-bytes | with-output-to-string | would-be-future | wrap-evt | wrapped-extra-arg-arrow-extra-neg-party-argument | wrapped-extra-arg-arrow-real-func | wrapped-extra-arg-arrow\\? | writable<%> | write | write-byte | write-bytes | write-bytes-avail | write-bytes-avail\\* | write-bytes-avail-evt | write-bytes-avail\\/enable-break | write-char | write-special | write-special-avail\\* | write-special-evt | write-string | writeln | xor | zero\\?\n) (?=$|[()\\[\\]{}\",'`;\\s])",
```

Comment:
Shouldn't we expand this line? It's really hard to find out what's changed on GitHub

Reply:
You surely can.